package.json

```diff
@@ -27,15 +27,15 @@
   ],
   "main": "index.js",
   "engines": {
-    "node": ">=6.4.1"
+    "node": ">= 14.0.0"
```

Comment:
I'm not sure why we need to update this.
Isn't `mocha` a devDependency? Do we really need to update this?

Reply:
I'm a little confused about this part, too.
If you agree, I will revert this part.

Reply:
I've tested with node 12 for `npm run test` and it's running well. Would you please also test it?
I guess if it's working well, we don't need to update the minimum node version.

Reply:
`npm run test` works fine on node 12, so I will revert this.